rust/sbp2json/src/bin/json2json.rs

```diff
@@ -12,10 +15,20 @@ static GLOBAL: mimalloc::MiMalloc = mimalloc::MiMalloc;
 ///
 /// Typical usage:
 ///
+///     json2json --input [PATH] --output [PATH]
+///
 ///     cat console-json-log.json | json2json
 #[derive(Debug, Parser)]
 #[clap(name = "json2json", verbatim_doc_comment)]
 struct Options {
+    /// Path to input file
+    #[clap()]
```

Comment:
I think you can omit the whole attribute `#[clap()]` if you're not passing anything to it